components/newsroom/Newsroom.js

```diff
@@ -10,7 +10,7 @@ import TextLink from '../typography/TextLink'
 export default function Newsroom() {
   return (
     <> 
-      <div className="text-center">
+      <div className="text-center mt-16">
```

Comment:
```suggestion
      <div className="text-center mt-12">
```

Sorry for making changes again but you have to follow the spacing according to other headings too in the website.